components/AppCopyright.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div>
+    <v-text font="sans" size="1.25rem"> 2021 Arcana Technologies Ltd. </v-text>
+  </div>
+</template>
+
+<script>
+import VText from './lib/VText.vue'
+export default {
+  components: { VText },
+}
+</script>
+
+<style lang="postcss" scoped>
+div {
+  padding: 2.5rem 0;
+  text-align: center;
+}
+</style>
```

Comment:
A few related changes:

- **Copyright text** Add missing "Pvt" to the copyright text. Determine the current year using a computed property.
- **Import** Use a root-relative path for the import. Remove the unnecessary `.vue` extension. Add a new line after the imports.
- **Styling** Style the `v-text` directly using a class instead of using a wrapper div. Set padding to `2.5rem` on all sides.

```
<template>
  <v-text class="copyright" font="sans" size="1.25rem">
    {{ currentYear }} Arcana Technologies Pvt Ltd.
  </v-text>
</template>

<script>
import VText from '~/components/lib/VText'

export default {
  components: { VText },
  computed: {
    currentYear() {
      return new Date().getFullYear()
    },
  },
}
</script>

<style lang="postcss" scoped>
.copyright {
  display: block;
  padding: 2.5rem;
  text-align: center;
}
</style>
```